index.js

```diff
@@ -1901,6 +1901,9 @@
   //.
   //. > reduce (concat, '', Cons ('foo', Cons ('bar', Cons ('baz', Nil))))
   //. 'foobarbaz'
+  //.
+  //. > reduce (concat, '', {foo: 'x', bar: 'y', baz: 'z'})
+  //. 'yzx'
```

Comment:
What do you think of this example, @kapooostin?

Thank you very much for submitting this pull request, by the way. :clap:

Reply:
Great, thanks you! Now it is illustrative enough without unnecessary complications.